content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/embeddable/youtube/_cq_dialog/.content.xml

```diff
@@ -38,6 +38,84 @@
             fieldDescription="Height of the YouTube video player."
             fieldLabel="Height"
             name="./youtubeHeight"/>
+        <parameters
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="granite/ui/components/coral/foundation/container"
+            fieldDescription="Parameters of the YouTube video player."
+            fieldLabel="Parameters">
+            <items jcr:primaryType="nt:unstructured">
+                <mute
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                    text="Enable Mute"
+                    checked="${cqDesign.youtubeMuteDefaultValue == 'true'}"
+                    uncheckedValue="false"
+                    value="true"
+                    fieldDescription="This parameter specifies whether the video will play muted by default. Enabling this increases the chance that Autoplay works in modern browsers."
+                    name="./youtubeMute">
+                    <granite:rendercondition
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/foundation/renderconditions/simple"
+                        expression="${cqDesign.youtubeMuteEnabled == 'true'}"/>
+                </mute>
+                <autoplay
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                    text="Enable Autoplay"
+                    checked="${cqDesign.youtubeAutoPlayDefaultValue == 'true'}"
+                    uncheckedValue="false"
+                    value="true"
+                    fieldDescription="This parameter specifies whether the initial video will automatically start to play when the player loads. This is only effective on Publish or when using &quot;View as Published&quot;."
+                    name="./youtubeAutoPlay">
+                    <granite:rendercondition
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/foundation/renderconditions/simple"
+                        expression="${cqDesign.youtubeAutoPlayEnabled == 'true'}"/>
+                </autoplay>
+                <loop
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                    text="Enable Loop"
+                    checked="${cqDesign.youtubteLoopDefaultValue == 'true'}"
```

Comment:
Due to the typo ( should be `youtubeLoopDefaultValue `instead of `youtub**t**eLoopDefaultValue`), the default value is not set property in the edit dialog.

Reply:
Fixed in https://github.com/adobe/aem-core-wcm-components/pull/1301/commits/86df8b09bbad467797d30dbc05f87dc90d360de9.